check_tags_in_requests.py

```diff
@@ -106,8 +109,34 @@ def checkTagInRequest(self, req, a):
 
         return True
 
+    def isNewPackage(self, tgt_project, tgt_package):
+        try:
+            osc.core.show_devel_project(self.apiurl, tgt_project, tgt_package)
+        except HTTPError:
```

Comment:
throws HTTPError undefined for me. Also, show_package_meta would be more suitable here